Dockerfiles/agent/Dockerfile

```diff
@@ -21,32 +24,39 @@ RUN apt-get update \
 #   - static libraries                   # FIXME: move upstream
 #   - jmxfetch on nojmx build
 RUN dpkg -x /datadog-agent*_amd64.deb . \
- && echo "Cleanup and trimming" \
  && rm -rf usr etc/init lib \
     opt/datadog-agent/sources \
     opt/datadog-agent/embedded/share/doc \
     opt/datadog-agent/embedded/share/man \
  && find opt/datadog-agent/ -iname "*.a" -delete \
  && if [ -z "$WITH_JMX" ]; then rm -rf opt/datadog-agent/bin/agent/dist/jmx; fi \
- && echo "Linking embedded ssl certificates" \
- && ln -s /opt/datadog-agent/embedded/ssl etc/ssl
+ && ln -s /opt/datadog-agent/embedded/ssl etc/ssl \
+ && mkdir conf.d checks.d
```

Comment:
With https://github.com/DataDog/datadog-agent/pull/1202 do we still need the mkdir here?

Reply:
This is creating `/conf.d` and `/checks.d` from where the custom configs/checks are copied by the entrypoint. We used to declare them as volumes, but that creates 2 empty docker volumes per container, even if we don't use them.
Creating the folders for discoverability by the users, so they have visual confirmation they need to inject custom things there.